app.js

```diff
@@ -87,7 +87,7 @@ function initSession(req, res, next) {
     cookie: {
       secure: (req.protocol === 'https')
     },
-    key: 'hmbrp.sid',
+    key: config.session.key,
```

Comment:
Looks as if it might now be called `name` https://github.com/expressjs/session/blob/master/index.js#L89